services/app/src/components/views/quest/cardtemplates/combat/Victory.test.tsx

```diff
@@ -1,6 +1,100 @@
+import {configure, render} from 'enzyme';
+import * as React from 'react';
+import Victory, {Props} from './Victory';
+import {MAX_ADVENTURER_HEALTH} from 'app/Constants';
+import {resolveCombat} from '../Params';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
+
+const HEAL_SUBSTR = 'All adventurers heal';
+const LOOT_SUBSTR = 'Draw Loot';
+const LEVEL_SUBSTR = 'LEVEL UP!';
+const HORROR_SUBSTR = 'The Horror';
+const FUTURE_SUBSTR = 'The Future';
+
 describe('Combat victory', () => {
-  test.skip('shows a victory page', () => { /* TODO */ });
-  test.skip('shows healing if not suppressed', () => { /* TODO */ });
-  test.skip('shows loot if not suppressed', () => { /* TODO */ });
-  test.skip('shows levelup if not suppressed', () => { /* TODO */ });
+  const TEST_VP: VictoryParameters = {
+    heal: MAX_ADVENTURER_HEALTH,
+    loot: true,
+    xp: true,
+  };
+
+  function setup(overrides?: Partial<Props>) {
+    const props: Props = {
+      combat: {
+        ...resolveCombat(null),
+        custom:false,
+        levelUp: true,
+        loot: [],
+      },
+      victoryParameters: {...TEST_VP},
+      settings: {
+        showHelp: true,
+        contentSets: {
+          horror: false,
+          future: false,
+        },
+      },
+      theme: 'dark';
+      onCustomEnd: jasmine.createSpy('onCustomEnd'),
+      onEvent: jasmine.createSpy('onEvent'),
+      ...overrides,
+    };
+    return {props, e: render(<Victory {...(props as any as Props)} />, undefined /*renderOptions*/)};
+  }
+
+  test('shows a victory page', () => {
+    expect(setup().e.text()).toContain('Victory');
+  });
+  test('shows healing if healing given', () => {
+    expect(setup({
+      victoryParameters: {..TEST_VP, heal: MAX_ADVENTURER_HEALTH}
+    }).e.text()).toContain(HEAL_SUBSTR);
+  });
+  test('hides healing if suppressed', () => {
+    expect(setup({
+      victoryParameters: {..TEST_VP, heal: 0}
+    }).e.text()).not.toContain(HEAL_SUBSTR);
+  });
+  test('shows loot if loot given', () => {
+    expect(setup({
+      victoryParameters: {..TEST_VP, loot: true},
+      combat: {
+        ...resolveCombat(null),
+        loot: [{count: 1, tier: 1}],
+      },
+    }).e.text()).toContain('One tier I loot');
+  });
+  test('hides loot if loot suppressed', () => {
+    expect(setup({
+      victoryParameters: {..TEST_VP, loot: false}
+    }).e.text()).not.toContain(LOOT_SUBSTR);
+  });
+  test('shows levelup if not suppressed', () => {
```

Comment:
This test is technically "show levelup if not suppressed... and the party should be leveling up", right? Since even when not suppressed, parties don't level up after every fight